### PR TITLE
Updated the network and related website links post space race launch.

### PR DESCRIPTION
**Pull request by jennijuju**

### CODE REVIEW

File: docs/mine/spacerace.md [hunk 1]
```diff
@@ -49,15 +49,23 @@ The top 50 miners in each region and globally are eligible to split a reward poo
...
-The top 20 FIL-denominated block reward producers who are also eligible to receive Space Race rewards will share a reward pool of 100k FIL on a pro rata basis. For example, if you receive 100,000 FIL in block rewards, and the top 20 producers cumulatively receive 2,000,000 FIL, you would be eligible to receive an additional 5,000 FIL. Like other competition rewards, any FIL received will vest linearly over six months. During the calibration period, miners can check their total block rewards mined at https://reward.calibration.fildev.network.
+The top 20 FIL-denominated block reward producers who are also eligible to receive Space Race rewards will share a reward pool of 100k FIL on a pro rata basis. For example, if you receive 100,000 FIL in block rewards, and the top 20 producers cumulatively receive 2,000,000 FIL, you would be eligible to receive an additional 5,000 FIL. Like other competition rewards, any FIL received will vest linearly over six months. During the calibration period, miners can check their total block rewards mined at the [reward dashboard](https://reward.testnet.filecoin.io/d/P_Q13JnMz/top-miners-by-block-reward?orgId=1&refresh=25s&from=now-30m&to=now&kiosk).
```
[the remark this answers is not in the document]
Comment (mishmosh) [reply docs/mine/spacerace.md:52]: To disambiguate from some instances where we call the storage reward pool the "competition rewards".

File: docs/mine/spacerace.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ A miner’s “location,” for regional leaderboards, is the physical location
 
 ## How do I participate?
 
-**The calibration period is currently open.** To join, run at least 1 miner on the [calibration devnet](https://docs.filecoin.io/how-to/networks/#calibration-devnet). A preview of calibration standings is available on the [Preview Dashboard](https://calibration.spacerace.filecoin.io/). 
+To join, run at least 1 miner on the [testnet](https://docs.filecoin.io/how-to/networks/#testnet). A preview of Space Race standings is available on the [Dashboard](https://spacerace.filecoin.io/).
```
Comment (mishmosh) [inline docs/mine/spacerace.md:18]: This paragraph was originally for the calibration period. Now that it's over, let's delete this para entirely and move the dashboard link to the competition section.

```suggestion
```

File: docs/mine/spacerace.md [hunk 3]
```diff
@@ -49,15 +49,23 @@ The top 50 miners in each region and globally are eligible to split a reward poo
...
-The [calibration devnet](https://docs.filecoin.io/how-to/networks/#calibration-devnet) is designed for the initial calibration phase in order to prepare equipment and client setups. Once the competition starts, the [testnet](https://docs.filecoin.io/how-to/networks/#testnet) will be the used network.
+The [testnet](https://docs.filecoin.io/how-to/networks/#testnet) is the used network.
+
+#### Where can I get test FIL?
+
+You can get FILs from the [faucet](https://spacerace.faucet.glif.io).
+
+#### Where can I find the stats of the network and progress of Space Race?
+
+Network stats for the testnet can be found on the [dashboard](https://stats.testnet.filecoin.io/d/z6FtI92Zz/chain?orgId=1&refresh=45s&from=now-30m&to=now&kiosk), and Space Race standings is available on [this dashboard](https://spacerace.filecoin.io).
```
Comment (mishmosh) [inline docs/mine/spacerace.md:68]: At this point we have 3 things referred to as dashboards :( so let's be SUPER clear about each one: Competition Dashboard, Block Reward Dashboard, and Network Stats.

```suggestion
The [Competition Dashboard](https://spacerace.filecoin.io/) is where you can track Space Race storage progress and register your miner(s).

Block Rewards standings are listed on the [Block Rewards Dashboard](https://reward.testnet.filecoin.io/), and network stats on the [Testnet Network Stats](https://stats.testnet.filecoin.io/) page.
```

Comment (jennijuju) [reply docs/mine/spacerace.md:66]: thanks! i was a bit confused about them as well actually

File: docs/mine/spacerace.md [hunk 4]
```diff
@@ -15,7 +15,7 @@ A miner’s “location,” for regional leaderboards, is the physical location
 
 ## How do I participate?
 
-**The calibration period is currently open.** To join, run at least 1 miner on the [calibration devnet](https://docs.filecoin.io/how-to/networks/#calibration-devnet). A preview of calibration standings is available on the [Preview Dashboard](https://calibration.spacerace.filecoin.io/). 
+To join, run at least 1 miner on the [testnet](https://docs.filecoin.io/how-to/networks/#testnet). A preview of Space Race standings is available on the [Dashboard](https://spacerace.filecoin.io/). 
 
 **The competition period will begin Monday, August 24th at 22:00 UTC** and is open for 3 weeks. To participate in the competition, run 1 or more miners on the Testnet. You'll also need to complete these steps to be eligible for rewards:
```
[the remark this answers is not in the document]
Comment (jennijuju) [reply docs/mine/spacerace.md]: updated

**Pull request merged**